src/jrd/names.h

```diff
@@ -441,3 +441,8 @@ NAME("MON$FILE_ID", nam_mon_file_id)
 NAME("MON$GUID", nam_mon_guid)
 NAME("MON$NEXT_ATTACHMENT", nam_mon_na)
 NAME("MON$NEXT_STATEMENT", nam_mon_ns)
+
+NAME("RDB$BLOB_UTIL_HANDLE", nam_butil_handle)
+NAME("RDB$BLOB", nam_blob)
+NAME("RDB$VARBINARY_MAX", nam_varbinary_max)
+NAME("RDB$LONG_NUMBER", nam_long_number)
```

Comment:
Correct for `RDB$BLOB_UTIL_HANDLE`.

For `RDB$LONG_NUMBER` I want to create a generic domain as I think it does not make sense to create one domain for `OFFSET` and another for `LENGTH`. For `MODE` it would make sense, but I'm just reusing it there too. I now renamed it to `RDB$INTEGER`.